playbooks/archivematica-xenial/README.md

```diff
@@ -0,0 +1,59 @@
+# Archivematica playbook
+
+The provided playbook installs Archivematica on a local vagrant virtual
+machine. For instructions on using deploy-pub to install Archivematica on a
+Digital Ocean droplet, see the [Digital Ocean Droplet
+Deploy](docs/digital-ocean-install-example.rst) document.
+
+## Requirements
+
+- Vagrant 1.7 or newer
+- Ansible 2.2 or newer
+
+## How to use
+
+1. Download the Ansible roles:
```

Comment:
Tip: if your code example fits in one line you can indent with four spaces instead of using the triple backtick. The advantage is that your one-liner will fit into one line instead of three.